packages/openapi-react-query/test/fixtures/mock-server.ts

```diff
@@ -0,0 +1,124 @@
+import {
+  http,
+  HttpResponse,
+  type JsonBodyType,
+  type StrictRequest,
+  type DefaultBodyType,
+  type HttpResponseResolver,
+  type PathParams,
+  type AsyncResponseResolverReturnType,
+} from "msw";
+import { setupServer } from "msw/node";
+
+/**
+ * Mock server instance
+ */
+export const server = setupServer();
+
+/**
+ * Default baseUrl for tests
+ */
+export const baseUrl = "https://api.example.com" as const;
+
+/**
+ * Test path helper, returns a an absolute URL based on
```

Comment:
nitpick (these are ignorable, insignificant suggestions; just pointing out)

```suggestion
 * Test path helper, returns an absolute URL based on
```

Reply:
Fixed here and in `openapi-fetch`. I think in a near future, it would be great to move the mock-server outside to share it across libraries as I literally duplicated it.